state-sync/src/coordinator.rs

```diff
@@ -1302,14 +1279,10 @@ impl<T: ExecutorProxyTrait> StateSyncCoordinator<T> {
             TargetType::Waypoint(waypoint_version)
         } else {
             match self.sync_request.as_ref() {
-                None => {
-                    TargetType::HighestAvailable {
-                        // here, we need to ensure pending_ledger_infos is up-to-date with storage
-                        // this is the responsibility of the caller of send_chunk_request
-                        target_li: self.pending_ledger_infos.target_li(),
-                        timeout_ms: self.config.long_poll_timeout_ms,
-                    }
-                }
+                None => TargetType::HighestAvailable {
+                    target_li: None,
```

Comment:
is it bad to have a single target? I do feel a little mixed about completely eliminating any target.